project/SbtLauncherPlugin.scala

```diff
@@ -0,0 +1,50 @@
+import sbt._
+import Keys._
+
+
+object SbtLauncherPlugin extends AutoPlugin {
```

Comment:
eventaully.  This is step one.  Time-wise I'm way over budget for this week on this patch, as it already spiralled.   Let's add that as a todo for later, rather than yak-shaving now.